php_sql/electives.php

```diff
@@ -0,0 +1,69 @@
+<?php
+define("MAX_TITLE_LENGTH", 128, true);
+define("MAX_TEACHER_LENGTH", 1024, true);
+define("MIN_DESCRIPTION_LENGTH", 10, true);
+
+$host = "localhost";
+$username = "root";
+$pass = "";
+$dbname = "test";
+
+function validateTitle($title, &$errors)
+{
+  if (!$title) {
+    $errors['title'] = "Името е задължително поле.";
+  } elseif (strlen($title) > MAX_TITLE_LENGTH) {
+    $errors['title'] = "Името има максимална дължина 150 символа.";
+  }
+  if (array_key_exists('title', $errors)) {
+    echo $errors['title'];
+  }
+}
+
+function validateTeacher($teacher, &$errors)

```

Comment:
Can you see a way of combining the different validation functions into a single one that can be reused?